tests/api_dataset_restful/test_dataset_verify.py

```diff
@@ -30,18 +35,20 @@ async def test_dataset_verify_pre_should_return_200(client, httpx_mock, dataset)
         json=payload,
     )
     assert res.status_code == 200
-    assert res.json() == {'code': 200, 'error_msg': '', 'num_of_pages': 1, 'page': 0, 'result': {}, 'total': 1}
+    assert res.json() == {'result': {'any': 'any'}}
 
 
-async def test_get_bids_msg_should_return_error_when_exception_happens(client, httpx_mock):
+async def test_get_bids_msg_should_not_found_should_return_404(client, test_db):
     dataset_code = 'utdataset'
     res = await client.get(f'/v1/dataset/bids-msg/{dataset_code}')
-    assert res.status_code == 500
-    assert 'Psql Error' in res.json()['result']
+    assert res.status_code == 404
+    assert res.json() == {'error': {'code': 'global.not_found', 'details': 'Requested resource is not found'}}
 
 
-async def test_get_bids_msg_should_return_200(client, httpx_mock, test_db, bids_results):
+async def test_get_bids_msg_should_return_200(client, bids_results):
     dataset_code = 'utdataset'
     res = await client.get(f'/v1/dataset/bids-msg/{dataset_code}')
+    expected_bids_result = json.loads(BIDSResultSchema.from_orm(bids_results).json())
```

Comment:
Once you use `BaseSchema` base class for `BIDSResultSchema` then you can change here to:
```suggestion
    expected_bids_result = BIDSResultSchema.from_orm(bids_results).to_payload()
```